package.json

```diff
@@ -211,7 +211,8 @@
                                 "^\"\\${workspaceFolder}/lambda/index.js\"",
                                 "--region",
                                 "${1|NA,EU,FE|}"
-                            ]
+                            ],
+                            "cwd": "^\"\\${workspaceFolder}/lambda\""
```

Comment:
I guess VSCode already handles this gracefully, but did we check the fix to be working as expected in windows as well?

Reply:
yup!